### ISSUE TITLE
A few open to public, paid "gaming booster service" providers operate in cn region. I wonder how they bypassed the WaII

### ISSUE THREAD

**Issue by gitporras** [net4people/bbs#347]
and is it a potential target to mimic traffic? also should I be concerned about safety when using those?

I am a newbie in tunnel apps here.

In 2022-23 I saw posts almost every month poping up in cn gaming bbs sharing names of these service providers,
And these apps can make it possible to play online games operating in global section (their servers always locate in JP,KR,SG,or US, not able to connect before), 
When I installed one of these apps and ran them, there was a "key icon" on my phone status bar, I think it woeks as a VPN?
Some of them even allow access to PlayStore although some of these tools show notices writes "we don't provide access for stores anymore" ,

In risky time like early March 2024, these tools also worked well (from my experience)
If friends here haven't haerd them I'll give some links talking about that, (cn website warning)
(艾澤拉斯地理論壇)
(address of that bbs) /read.php?tid=35438541
(address of that bbs) /read.php?tid=38985126
(address of that bbs) /read.php?tid=38705145

You may not play these games being discussed but surely you have heard of, or have a friend who plays,
Users using these apps should be less than wechat video users, Bittorent clients but not too little either

Also online games need large data download to update (100 MB to 5 GB or more),
and these "gaming booster" apps helped downloading through their channels (I think large data flow is allowed by the WaIl?)

(I'd like to add that, very frequent user download actions only happen on certain day within a month or two months, when a game have a mandatory update to let user reinstall a game and have large data update, if you played WoW before you should remember such a thing)

Above are the features which make these apps possible targets to use as disguise, although I am not even a coding learner to judge

2. This part are the risks ,
These "gaming booster apps" always ask for a phone number to register, and want resident name and code to input
I'm not sure these tools send personal information to the WaIl regularly

Gaming data centers have a fixed range in IP and domain, and games won't be popular very long time like apple/MS/Github website
(WoW ran 14 years and shut down in cn, Overwatch popularity droped much faster than that)

So if friends here want to discover of what encryption these tools are using, better ask a friend to test for you
phone number, payment detail, resident information used, phone IMEI, too much information is open

And also I have questions off topic,

Now I use these booster apps for a longer time every week (need to play PVP like many hours)
Will I be marked a potential person to drink teа just because there is data in and out in geoip cn (if game global data center in ShεnZhεn and I use a proxy outside to connect, creates data flow back inside the WaIl then trace me, although data centers may be in SG, HK not cn)

Second, I saw people talking about "cIash" (even in threads inside WaIl ) many times, is that a project going to be killed like SSR soon, or not yet?

**Comment by RPRX** [comment]
热知识：在中国，有一些“没有墙”的“专线”供政府、学术机构（比如大学）、愿意出高价的人（比如大型企业）使用

**Comment by wkrp** [comment]
> 热知识：在中国，有一些“没有墙”的“专线”供政府、学术机构（比如大学）、愿意出高价的人（比如大型企业）使用
> 
> _Hot Knowledge: In China, there are "private lines" without walls for the government, academic institutions (e.g., universities), and those willing to pay high prices (e.g., large corporations)._

Yes, that's my understanding. There's some discussion of high-speed transnational gateways in the ["Great Bottleneck" paper](https://censorbib.nymity.ch/#Zhu2020a) (previous discussion #96). I presumed that the gaming booster services paid more money for higher-quality links.

https://censorbib.nymity.ch/pdf/Zhu2020a.pdf#page=19
https://meetbot.debian.net/tor-meeting/2021/tor-meeting.2021-10-28-16.00.log.html#l-172

> According to a recent report by China Academy of Information and Communications Technology, all the three state-own ISPs have set up a premium transnational network (primarily for business uses) to maximize their profit. For example, China Telecom Global’s official website explicitly claims four tiers of services to connect to Chinese users. (1) China Access, (2) ChinaNet Paid-Peer, (3) Global Transit (GT), (4) Global Internet Access (GIA). Basically, the first three share the same point-of-presence or international gateway and therefore similar potential bottleneck, while Global Internet Access has a different dedicated CN2 international gateway.

Here's an archived page about China Telecom's tiers:
https://web.archive.org/web/20230420180335/https://www.chinatelecomglobal.com/expertise?category=product-and-services&subcategory=internet&pid=gis

**Comment by UjuiUjuMandan** [comment]
> _Hot Knowledge: In China, there are "private lines" without walls for the government, academic institutions (e.g., universities), and those willing to pay high prices (e.g., large corporations)._

I understand the performance would be better in these lines, but without walls? Probably not. Just dig into OONI data to see if any Chinese probe can access blocked foreign news sites.

**Comment by momigapi1990** [comment]
> any indication that the government even wants to block the games or the booster services at all?
Video games/mmo games are not priority targets to block access, at the same time they are still treated as publicated material out of the regime's reach, there can be unwanted content inside, (similar to Instagram, search engines etc.) not serious and not safe either
Example:
Soviets vs Alliance in Red Alert2
PLA army in an old RTS genre
reference of Covid, or mocking That Emperor using nickname or whatever way

And on top of that, global games always have google,FB login options to create user profiles, another reason these game publishers are not compatible, and the boost services have their day:
games supported are limtied=lower risk, and they are paying=tax and income

**Comment by Lanius-collaris** [comment]
> I understand the performance would be better in these lines, but without walls? Probably not. Just dig into OONI data to see if any Chinese probe can access blocked foreign news sites.

AS4538 (China Education and Research Network Center) :
https://explorer.ooni.org/chart/mat?probe_cc=CN&probe_asn=AS4538&since=2023-05-05&until=2024-05-06&time_grain=month&axis_x=measurement_start_day&test_name=web_connectivity
A large, persistent drop in OONI measurement coverage from AS4538. 🙃

https://www.jsdelivr.com/globalping
Globalping can also be used to test some domains. (Some censored domains can't trigger TCP RST bidirectionally)

### PoC
test type: HTTP
target: 101.6.15.130 (`mirrors.tuna.tsinghua.edu.cn`)
location: USA
limit: 5
host: `www.nytimes.com`
port: 443
protocol: HTTPS
method: GET

**Comment by wkrp** [comment]
> AS4538 (China Education and Research Network Center) :
> https://explorer.ooni.org/chart/mat?probe_cc=CN&probe_asn=AS4538&since=2023-05-05&until=2024-05-06&time_grain=month&axis_x=measurement_start_day&test_name=web_connectivity
> A large, persistent drop in OONI measurement coverage from AS4538. 🙃

BTW the cause of the drop in OONI measurements from China since 2023-07 is known. See:
* #270

**Comment by Lanius-collaris** [comment]
> BTW the cause of the drop in OONI measurements from China since 2023-07 is known. See:
> 
>     * [OONI blocking (website and backend infrastructure) in China since 2023-07-07 #270](https://github.com/net4people/bbs/issues/270)

@wkrp
The point is, blocking of OONI seems to impact AS4538. A few years ago, some people told me the China Education and Research Network had "privilege".
